Modules/Video/IO/include/itkVideoFileWriter.hxx

```diff
@@ -253,7 +253,7 @@ VideoFileWriter<TInputVideoStream>::InitializeOutputParameters()
   const VideoStreamType * input = this->GetInput();
 
   // Set dimensions
-  m_Dimensions.empty();
+  m_Dimensions.clear();
```

Comment:
> .clear() only returns whether the container is empty or not.

You mean "`.empty()` only returns whether the container is empty or not."

Reply:
Thanks, fixed.